src/ext/oblivc/ot.c

```diff
@@ -1033,6 +1080,69 @@ bcipherCryptNoResize(BCipherRandomGen* gen,const char* key,int nonce,
   randomizeBuffer(gen,dest,n);
   memxor(dest,src,n);
 }
+
+
+/* 
```

Comment:
Ok, this is starting to get on my OCD :) It just so happens that recent versions of git highlights trailing whitespaces, and you tend to leave one after each "/*" comment starts.

Reply:
Hmm. I wonder how this happens. I am not doing it consciously.